### DISCUSSION TITLE
Bug in auto render view

### DISCUSSION THREAD

**Discussion by NVD-R** [livewire/livewire#4386] [7. Bugs (archived)]
(https://github.com/livewire/livewire/issues/4379)
when you change `view_path `in config component auto render not worked and need changes `livewire/src/Component.php` file in line `172`. you should read `view_path `path from config.

**Comment by joshhanley** [comment]
@NVD-R thanks for reporting!

Can you explain in more detail what you mean by auto render not working and explain what currently is happening?

Can you also give us an example of what you were trying to change `view_path` to plus an example component so we can reproduce the issue?

Thanks!
